lib/python/platform_code_vocab.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Find the platform_code platform_name equivalence from poolparty platform vocab
+xml file stored in content.aodn.org.au
+
+How to use:
+    import os, sys
+    sys.path.insert(0, os.path.join(os.environ.get('DATA_SERVICES_DIR'), 'lib'))
+    from python.platform_code_vocab import *
```

Comment:
Just pointing it out again, I don't think this is a good way to set this up. Just because you can do it doesn't mean you should. The idea of environment variables is that you can change them and then run the same code in a different environment. You can't do that if some of your environment is hard-coded in.

All you need to do is `export PYTHONPATH=$DATA_SERVICES_DIR/lib/python`. This is already done for the `projectofficer` user in prod. In the PO box you could just add it to the `env` file, or at the top of your `incoming_handler.sh` script. Then you can just say
```python
from platform_code_vocab import *
```